src/components/reportSummary/reportSummary.tsx

```diff
@@ -7,29 +7,52 @@ import {
 } from '@patternfly/react-core';
 import { css } from '@patternfly/react-styles';
 import React from 'react';
+import { InjectedTranslateProps, translate } from 'react-i18next';
+import { FetchStatus } from '../../store/common';
 import { styles } from './reportSummary.styles';
 
-interface ReportSummaryProps {
+interface ReportSummaryProps extends InjectedTranslateProps {
   title: string;
   subTitle?: string;
   children?: React.ReactNode;
   detailsLink?: React.ReactNode;
+  status: number;
 }
 
-const ReportSummary: React.SFC<ReportSummaryProps> = ({
+const ReportSummaryBase: React.SFC<ReportSummaryProps> = ({
   title,
   subTitle,
   detailsLink,
   children,
-}) => (
-  <Card className={css(styles.reportSummary)}>
-    <CardHeader>
-      <Title size="lg">{title}</Title>
-      {Boolean(subTitle) && <p className={css(styles.subtitle)}>{subTitle}</p>}
-    </CardHeader>
-    <CardBody>{children}</CardBody>
-    {Boolean(detailsLink) && <CardFooter>{detailsLink}</CardFooter>}
-  </Card>
-);
+  status,
+  t,
+}) =>
+  status === FetchStatus.inProgress ? (
+    <Card className={css(styles.reportSummary)}>
+      <CardHeader>
+        <Title size="lg">{title}</Title>
+        {Boolean(subTitle) && (
+          <p className={css(styles.subtitle)}>{subTitle}</p>
+        )}
+      </CardHeader>
+      <CardBody>
+        {t('loading')}
+        ...
+      </CardBody>
+    </Card>
+  ) : (
+    <Card className={css(styles.reportSummary)}>
+      <CardHeader>
+        <Title size="lg">{title}</Title>
+        {Boolean(subTitle) && (
+          <p className={css(styles.subtitle)}>{subTitle}</p>
+        )}
+      </CardHeader>
+      <CardBody>{children}</CardBody>
+      {Boolean(detailsLink) && <CardFooter>{detailsLink}</CardFooter>}
```

Comment:
Would it be possible to change just the card body instead of creating a separate card, header, and body? Not certain what's the best practice for React, but otherwise looks identical?